pagopa/io-app
# fix: [IOBP-42] Removal of privative methods from UI

## Short description
removed privative methods renderers from wallet home and add_payment_method screens 

## List of changes proposed in this pull request
- wallet list's filter returns null in case of a privative method
- wallet_add is now missing the privative method entry

## How to test
`yarn jest`, also navigate to wallet home screen with a privative method onboarded (with the dev-server you can add any),  and to the add method screen, and make sure there is no "privative method" entry
